docs/concepts.md

```diff
@@ -1,37 +1,37 @@
 Concepts
 ========
 
-Simplistic Overview of request processing
+Overview of request processing
 -----------------------------------------
 
-A request is received on the edge server (fastly) and is associated to a service configuration.
-The configuration contains a varnish cache script (VCL) that extracts processing information from
-the request. those are:
+An HTTP request is received on the edge server (fastly) and is associated to a [service configuration][fastly-services].
+The configuration contains a varnish cache script ([VCL][vcl]) that extracts processing information from
+the request. Those are:
 
 - requested resource
 - selector and extension
 - strain name
 
-based on this it looks up:
+Based on this it looks up:
 
 - backend type and url
 - content repository
 - adobeioruntime action or static server url
```

Comment:
As someone relatively new to helix, it wasn't clear to me that a static server was even a part of the equation until I read this line. Perhaps we need to explain the static server somewhere in these docs.

Reply:
This is likely outdated information: today as we know, static content gets proxied through the static action...